src/game_api/state.hpp

```diff
@@ -72,7 +72,7 @@ struct StateMemory
     uint32_t time_last_level;
     uint32_t time_level;
     int32_t ia00;
-    int32_t ia04;
+    uint32_t money_total;

```

Comment:
So it appears this won't be the total money that appears in the top right of the screen but the money from the previous levels. I wonder if it would be possible to name this something else or provide additional documentation. Maybe would be less confusing if this was named `money_last` to note that is the money total at the end of the last level.

Reply:
Sure, I've named it money_last_levels, to make it slightly more descriptive